hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java

```diff
@@ -243,29 +250,41 @@ public void testFsActionALL() throws Exception {
         isCheckAccessEnabled);
     Path testFilePath = setupTestDirectoryAndUserAccess("/test8.txt",
         FsAction.ALL);
-    assertTrue(isAccessible(testFilePath, FsAction.EXECUTE));
-    assertTrue(isAccessible(testFilePath, FsAction.WRITE));
-    assertTrue(isAccessible(testFilePath, FsAction.WRITE_EXECUTE));
-    assertTrue(isAccessible(testFilePath, FsAction.READ));
-    assertTrue(isAccessible(testFilePath, FsAction.READ_EXECUTE));
-    assertTrue(isAccessible(testFilePath, FsAction.READ_WRITE));
-    assertTrue(isAccessible(testFilePath, FsAction.ALL));
+    assertAccessible(testFilePath, FsAction.EXECUTE);
+    assertAccessible(testFilePath, FsAction.WRITE);
+    assertAccessible(testFilePath, FsAction.WRITE_EXECUTE);
+    assertAccessible(testFilePath, FsAction.READ);
+    assertAccessible(testFilePath, FsAction.READ_EXECUTE);
+    assertAccessible(testFilePath, FsAction.READ_WRITE);
+    assertAccessible(testFilePath, FsAction.ALL);
+  }
+
+  private void assertAccessible(Path testFilePath, FsAction fsAction)
```

Comment:
this is good -thanks for doing this. if you ever find yourself duplicating code in tests, it's usually a sign to add your own assertions.

Reply:
Thanks :)